### PR TITLE
basic dropdown to pick location closes #66

### LINKED ISSUE #66: Create a drop-down element for room selection
We should change the locations from buildings to rooms on a dropdown element.

### PR DESCRIPTION
**Pull request by kerrd89**
on the admin page, a user sees a dropdown to select the location of choice.

### CODE REVIEW

File: lib/components/AdminSpike.js [hunk 1]
```diff
@@ -37,24 +37,12 @@ const AdminSpike = ({ spike, createSpike, updateSpike, deleteSpike }, key ) => {
...
-            type='radio'
-            name='location'
-            value='Blake'
-            onClick={(e)=>updateSpike(spike, 'location', e.target.value)}
-          /> Blake
+          <select name='location' onChange={(e)=>updateSpike(spike, 'location', e.target.value)}>
+            <option value='none'>none</option>
+            <option value='GBank'>GBank</option>
+            <option value='Gusto'>Gusto</option>
+            <option value='Fartsville'>Fartsville</option>
```
Comment (swanie21) [inline lib/components/AdminSpike.js]: Do we have the actual classroom names/numbers with the map from Joanne? I think we may have numbers...

**Comment by Peter-Springer** [comment]
I can not see a drop down on this branch. How about anyone else?

**Comment by Jeff-Duke** [comment]
looks good to me, @Peter-Springer not sure why you're not seeing it.  are you in admin view?  

I like that we can change the room even after a spike has been approved.

**Comment by swanie21** [comment]
I'm able to see it in the admin spike card.

**Comment by Jeff-Duke** [comment]
I'm good with merging it if you are @swanie21

**Comment by swanie21** [comment]
I am and then hopefully Pete can pull down the master and it will be visible.

**Comment by Peter-Springer** [comment]
I see it on master weird

**Pull request merged**